docs/release_notes/ibexa_dxp_v4.6.md

```diff
@@ -0,0 +1,466 @@
+
+For more information about Remote PIM support and the solution's limitations, see [PIM product guide](pim_guide.md#limitations).
+
+#### Virtual products
+
+With this feature, you can create virtual products - non-tangible items such as memberships, services, warranties.
+To create a virtual product, first, you have to create a virtual product type.
+Virtual products don’t require shipment when they're purchased without other physical products.
+
+For more information, see [Create virtual products](https://doc.ibexa.co/projects/userguide/en/master/pim/create_virtual_product/).
+
+#### Product page URLs
+
+When you are creating a new product type, you can set up a product URL alias name pattern.
+With this feature, you can also create custom URL and URL alias name pattern field based on product attributes.
+Customized URLs are easier to remember, help with SEO optimization and reduce bounce rates on the website.
+
+For more information, see [Product page URLs](https://doc.ibexa.co/projects/userguide/en/master/pim/work_with_product_page_urls/).
+
+#### VAT assignment moved to a new place
+
+Users who are creating or editing a product type are less likely to forget about setting VAT rates, because they now have a more prominent place.
+
+![Assigning VAT rates to a product type](img/4.6_catalog_vat_rates.png)
+
+For more information, see [Create product types](https://doc.ibexa.co/projects/userguide/en/master/pim/create_product_types/).
+
+#### Updated VAT configuration
+
+VAT rates configuration has been extended to accept additional flags under the `extras` key.
+Developers can use them, for example, to pass additional information to the UI, or define special exclusion rules.
+
+For more information, see [VAT rates](https://doc.ibexa.co/en/master/pim/pim_configuration/#vat-rates).
+
+##### New Twig functions
+
+The `ibexa_is_pim_local` Twig helper has been introduced, which can be used in templates to [check whether product data comes from a local or remote data source](storefront_twig_functions.md#ibexa_is_pim_local), and adjust their behavior accordingly.
+Also, several new Twig functions have been implemented that help [get product availability information](product_twig_functions.md#ibexa_has_product_availability).
+
+##### New and modified query types
+
+The `ProductContentAwareListQueryType` has been created to allow finding products that come from a local database, while `ProductListQueryType` has been modified to find products from an external source of truth.
+
+##### New Search Criterion
+
+With `IsVirtual` criterion that searches for virtual or physical products, product search now supports products of virtual and physical type.
+
+#### Product migration
+
+[Product variants](importing_data.md#product-variants) and [product assets](importing_data.md#product-assets) can now be created through [data migration](data_migration.md).
+
+###  New features and improvements in Commerce [[% include 'snippets/commerce_badge.md' %]]
+
```

Comment:
We forgot about one very important feature for Commerce: **Integration with payment gateways** https://issues.ibexa.co/browse/IBX-5713